pkg/ast/structs.go

```diff
@@ -28,6 +28,7 @@ import (
 type QueryStruct struct {
 	SearchFilter *Node
 	PipeCommands *structs.QueryAggregators
+	IndexName    string
```

Comment:
is this field used somewhere ?

Reply:
Yes it will capture all the indexes passed as comma separated (just like how we pass in the json ind-0, ind-1). this was done so that we can reuse the `InitTableInfo` just like how we do it for a curl request.